.eslintrc.js

```diff
@@ -63,59 +63,57 @@ module.exports = {
                 ignoredNodes: ['JSXElement *']
             }
         ],
-        'key-spacing': [2, { beforeColon: false, afterColon: true }],
-        'keyword-spacing': [2, { before: true, after: true }],
-        'linebreak-style': 0,
-        'max-len': [2, 200, 2, { ignoreUrls: true, ignoreRegExpLiterals: true }],
-        'newline-after-var': 2,
-        'require-atomic-updates': 0,
-        'no-constant-condition': 2,
-        'no-dupe-class-members': 2,
-        'no-lonely-if': 2,
-        'no-multiple-empty-lines': [2, {
-            max: 1,
-            maxEOF: 0
-        }],
-        'no-prototype-builtins': 0,
-        'no-underscore-dangle': 2,
-        'no-var': 2,
-        'object-curly-spacing': [2, 'always'],
-        'object-curly-newline': [2, { consistent: true }],
-        'object-property-newline': [2, { allowAllPropertiesOnSameLine: true }],
-        'object-shorthand': [2, 'methods'],
-        'operator-linebreak': [0, 'before'],
-        'padded-blocks': [2, 'never'],
+        'key-spacing': ['error', { beforeColon: false, afterColon: true }],
+        'keyword-spacing': ['error', { before: true, after: true }],
+        'linebreak-style': ['error', 'unix'], // no carriage returns
+        'max-len': ['error', { code: 120 }], // be friendly to laptops
+        'newline-after-var': 'error',
+        'require-atomic-updates': 'warn', // okay to warn?
+        'no-constant-condition': 'error',
+        'no-dupe-class-members': 'error',
+        'no-lonely-if': 'error',
+        'no-multiple-empty-lines': ['error', { max: 1, maxEOF: 1 }], // let my people push enter!
+        'no-prototype-builtins': 'off', // why would we need this off?
+        'no-underscore-dangle': 'error',
+        'no-var': 'error',
+        'object-curly-spacing': ['error', 'always'], // kinda wide 
+        'object-curly-newline': ['error', { consistent: true }],
+        'object-property-newline': ['error', { allowAllPropertiesOnSameLine: true }],
+        'object-shorthand': ['error', 'methods'],
+        'operator-linebreak': ['off', 'before'],
+        'padded-blocks': ['error', 'never'],
+        // adds a lot of blank lines 🤔
         'padding-line-between-statements': [
```

Comment:
This one felt like excessive whitespace within a function

Reply:
There's a few conditions in here:

1. I think series of variable declarations doesn't need space.
2. not picky on space after variable declarations.
3. prefer for "return" to have a line